scripts/Projects.psm1

```diff
@@ -26,7 +26,9 @@ $Projects.Libraries = @(
     "Sarif",
     "Sarif.Converters",
     "Sarif.Driver",
-    "Sarif.WorkItems"
+    "Sarif.Multitool.Library"
```

Comment:
>Sarif.Multitool.Library [](start = 5, length = 23)

Will the multitool library be shipped in the Sarif.Sdk nuget package?

Reply:
No, it's a separate NuGet package `Sarif.Multitool.Library`.

---
In reply to: [483762094](https://github.com/microsoft/sarif-sdk/pull/2054#discussion_r483762094) [](ancestors = 483762094)